lib/signs/realtime.ex

```diff
@@ -101,17 +105,22 @@ defmodule Signs.Realtime do
       expiration_seconds: 130,
       read_period_seconds: 240,
       headway_stop_id: Map.get(config, "headway_stop_id"),
-      uses_shuttles: Map.get(config, "uses_shuttles", true)
+      uses_shuttles: Map.get(config, "uses_shuttles", true),
+      extra_audio_sign_ids: Map.get(config, "additional_signs", [])
     }
 
-    GenServer.start_link(__MODULE__, sign)
+    GenServer.start_link(__MODULE__, sign, name: String.to_atom(sign.id))
   end
 
   def init(sign) do
     schedule_run_loop(self())
     {:ok, sign}
   end
 
+  def get_sign_state(sign_id) do
```

Comment:
Yeah, it does add a bit of complexity but I think it's still very easy to reason about the interactions since Elixir is optimized for this sort of thing. And because the signs are just sharing their current state which is being updated once a second anyway, I think doing it this way actually isn't far off from refetching the predictions and doing the same calculations from the source data and is a bit more efficient if anything